pkg/internal/byteutils/byteutils.go

```diff
@@ -22,3 +22,12 @@ func LeftPadTo32Bytes(bytes []byte) ([]byte, error) {
 
 	return result, nil
 }
+
+// Reverse reverses bytes order in the slice.
```

Comment:
I would add information to the description that the function returns a new byte array without modifying the input array.